scripts/test-runner.js

```diff
@@ -257,10 +308,16 @@ function runTest(name, code, options, args) {
           }
         }
         if (markersIssue) break;
+        let codeToRun = addedCode + newCode;
+        if (args.es5) {
```

Comment:
Sounds like there is no need to lower using babel under es5 mode since we exclude es6 tests from running under es5 mode.